packages/common/src/content/fetchItemJobRecords.ts

```diff
@@ -0,0 +1,72 @@
+import HubError from "../HubError";
+import {
+  IHubDownloadJobRecord,
+  IHubJobRecord,
+  IHubJobRecordRequestOptions,
+  JobRecordType,
+} from "./types";
+
+/**
+ * Fetches job records for a specific item.
+ *
+ * @param id - The ID of the item.
+ * @param options - The options for fetching job records.
+ * @returns A promise that resolves to an array of job records.
+ * @throws {HubError} If any unimplemented options are provided.
+ */
+export async function fetchItemJobRecords(
```

Comment:
@tomwayson is the content folder the correct place for this function? Or should we create a `jobs` folder?

Reply:
Since the end point is under `/items` content seems reasonable to me, but I don't have strong feelings one way or the other (for once).

Reply:
I also don't have strong feelings, I'll keep it where it is. Thank you!